components/Custom/Button/index.tsx

```diff
@@ -30,7 +30,7 @@ const Button = ({
     <button
       {...props}
       className={[
-        'text-2xl px-8 py-2 rounded-3xl uppercase font-serif transform duration-100 hover:scale-105',
+        'text-2xl px-4 py-2 rounded-3xl uppercase font-serif transform duration-100 hover:scale-105 active:opacity-70 no-tap-highlight',
```

Comment:
The 'no-tap-highlight' could cause issues for colour blind people. 
I suggest copying over the "focus:" classes from the navbar buttons for this button as well...

Not too big of a problem though since the background is white and the opacity being lowered on click might just suffice.